circle.yml

```diff
@@ -7,12 +7,14 @@ dependencies:
     #decrypt encrypted credentials - https://github.com/circleci/encrypted-files/blob/master/circle.yml
     - openssl aes-256-cbc -d -in credentials.cipher -k $KEY >> ~/.circlerc
     - source ~/.circlerc
+    - npm install -g csslint
```

Comment:
would be better to have this dependancy in a `package.json` and cache the `node_modules` directory to speed up builds